deploy/ocm-12870/02-CronJob.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: sre-operator-reinstall
+  namespace: openshift-adp-operator
+spec:
+  ttlSecondsAfterFinished: 100
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "* * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: sre-operator-reinstall-sa
+          restartPolicy: Never
+          containers:
+          - name: operator-reinstaller
+            image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+            imagePullPolicy: Always
+            command:
+            - sh
+            - -c
+            - |
+              #!/bin/bash
+              set -euxo pipefail
+              NAMESPACE=openshift-adp-operator
+
+              # Check for the status OADP v1.3.4 version 
+              # (previous version: ocm-agent-operator.v1.2.x). 
+              # If we it's not succeeded, do OLM dance The resources will be re-created 
+              # with the latest version
+
+              CSV_PHASE=$(oc -n openshift-adp-operator get csv oadp-operator.v1.3.4 -ojsonpath='{.status.phase}{"\n"}') || true
```

Comment:
I think this won't work, as there will be 2 CSVs (one for 1.2 and one for 1.3 and I think both might have to be removed).